insights/specs/datasources/ls.py

```diff
@@ -36,50 +38,55 @@ def _list_items(spec):
                 ==============================================================
             """
             filters.append('_non_existing_')
-        return ' '.join(filters)
+        return filters
     raise SkipComponent
 
 
 @datasource(HostContext)
 def list_with_la(broker):
-    return _list_items(Specs.ls_la_dirs)
+    return ' '.join(_list_items(Specs.ls_la_dirs))
 
 
 @datasource(HostContext)
 def list_with_la_filtered(broker):
-    return _list_items(Specs.ls_la_filtered_dirs)
+    return ' '.join(_list_items(Specs.ls_la_filtered_dirs))
 
 
 @datasource(HostContext)
 def list_with_lan(broker):
-    return _list_items(Specs.ls_lan_dirs)
+    filters = set(_list_items(Specs.ls_lan_dirs))
+    if 'specs.fstab_mounted' in filters and FSTab in broker:
+        for mntp in broker[FSTab].mounted_on.keys():
+            mnt_point = os.path.dirname(mntp)
+            filters.add(mnt_point) if mnt_point else None
+    return ' '.join(sorted(list(filters)))
```

Comment:
@TZ3070  We can directly sort on a `set` type, this `list(filters)` converting can be skipped here.